packages/Organisms/SiteHeader/index.vue

```diff
@@ -332,9 +344,32 @@ export default {
       }
 
       img {
-        width: $rpl-site-header-logo-width;
+        width: 6.75rem;
         margin-left: $rpl-site-header-menu-toggle-border-spacing;
       }
+
+      &--vic-logo-static {
+        display: block; //always show vic.gov logo if no cobrand logo
+
+        img {
+          width: rem(88px);
```

Comment:
Please use a variable for this, also shouldn't the primary logo and the cobrand logo be the same size?

Reply:
Many sites actually specify custom CSS for their cobrand logo as the dimensions are slightly different. I've given the primary logo a set size of 98px so that it adheres to the the design guidelines.